var/spack/repos/builtin/packages/ngmlr/for_va_list.patch

```diff
@@ -0,0 +1,22 @@
+--- spack-src/src/ConvexAlignFast.cpp.bak	2017-06-27 02:01:12.000000000 +0900
++++ spack-src/src/ConvexAlignFast.cpp	2020-09-07 17:03:29.325279688 +0900
+@@ -8,7 +8,7 @@
+ #include <string.h>
+ #include <stdio.h>
+ #include <algorithm>
+-#include <x86intrin.h>
++#include <SSE2NEON.h>
```

Comment:
This patch is applied to all platforms, but sse2neon is only added for aarch64...